spec/models/user_spec.rb

```diff
@@ -3,95 +3,19 @@
 RSpec.describe(User, type: :model) do
   let(:user) { build(:user) }
 
-  context 'when name field is empty' do
-    it 'is invalid' do
-      user.name = ''
-      expect(user).to_not be_valid
-    end
-  end
+  it { is_expected.to validate_presence_of(:name) }
 
-  context 'when name field has number' do
-    it 'is invalid' do
-      user.name = 'B1b'
-      expect(user).to_not be_valid
-    end
-  end
+  it { is_expected.to_not validate_numericality_of(:name) }
 
-  context 'when name field is too short' do
-    it 'is invalid' do
-      user.name = 'Mi'
-      expect(user).to_not be_valid
-    end
-  end
+  it { is_expected.to validate_length_of(:name).is_at_least(3).is_at_most(30) }
 
-  context 'when name field is too long' do
-    it 'is invalid' do
-      user.name = 'Mikonikadjkrklfskdjfhdskjfhskjdfhsk'
-      expect(user).to_not be_valid
-    end
-  end
+  it { is_expected.to validate_presence_of(:surname) }
 
-  context 'when surname field is empty' do
-    it 'is invalid' do
-      user.surname = ''
-      expect(user).to_not be_valid
-    end
-  end
+  it { is_expected.to_not validate_numericality_of(:surname) }
```

Comment:
no need to validate that what we did not add

Reply:
@okohuch  done